stubs/Flask-Cors/flask_cors/decorator.pyi

```diff
@@ -10,10 +10,10 @@ LOG: Logger
 
 def cross_origin(
     *args: Any,
-    origins: str | list[str] | None = ...,
+    origins: str | Pattern[str] | list[str | Pattern[str]] | None = ...,
```

Comment:
`list[str | Pattern[str]]` is bad because list is invariant, so a `list[str]` wouldn't be accepted. The nicest solution would be if we can write `Sequence[str | Pattern[str]]`, since `Sequence` is covariant. Does it really require a list?

Reply:
Looking at the code, `Iterable` should suffice